### PR TITLE
Nitpicking abi-spec.rst

### PR DESCRIPTION
**Pull request by pirapira**

### CODE REVIEW

File: docs/abi-spec.rst [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
 ******************************************
 
-Basic design
+Basic Design
 ============
 
 The Application Binary Interface is the standard way to interact with contracts in the Ethereum ecosystem, both
 from outside the blockchain and for contract-to-contract interaction. Data is encoded following its type,
-according to this specification.
+according to this specification.  The types are not encoded.
```
Comment (axic) [inline docs/abi-spec.rst]: What do you mean by `types are not encoded`?

File: docs/abi-spec.rst [hunk 2]
```diff
@@ -334,10 +335,6 @@ would result in the JSON:
   "inputs": [{"name":"a","type":"uint256","indexed":true},{"name":"b","type":"bytes32","indexed":false}],
   "name":"Event2"
   }, {
-  "type":"event",
-  "inputs": [{"name":"a","type":"uint256","indexed":true},{"name":"b","type":"bytes32","indexed":false}],
-  "name":"Event2"
```
Comment (axic) [inline docs/abi-spec.rst:339]: I see this is duplicated?

File: docs/abi-spec.rst (hunk 1 above)
Comment (pirapira) [reply docs/abi-spec.rst]: No bytes in the call data say `uint8`.

File: docs/abi-spec.rst (hunk 2 above)
Comment (pirapira) [reply docs/abi-spec.rst:339]: That's why I'm removing one of the duplicates.

File: docs/abi-spec.rst (hunk 1 above)
Comment (axic) [reply docs/abi-spec.rst]: Wouldn't `The encoding is not self describing and thus requires a schema in order to decode.` be more descriptive?

Comment (pirapira) [reply docs/abi-spec.rst]: I like that.

File: docs/abi-spec.rst [hunk 3]
```diff
@@ -93,6 +93,7 @@ We distinguish static and dynamic types. Static types are encoded in-place and d
 * `string`
 * `T[]` for any `T`
 * `T[k]` for any dynamic `T` and any `k > 0`
+* `(T1,...,Tk)` if any `Ti` is dynamic for `1 <= i <= k`
```
Comment (chriseth) [inline docs/abi-spec.rst:96]: Good catch!

**Review by chriseth — APPROVED** [review]

**Pull request merged**